af_pipeline/archive/af_utils.py

```
@@ -0,0 +1,36 @@
+        Dict: renumbered residue dictionary
+    """
+
+    renumbered_res_dict = {}
+
+    for chain_id in res_dict:
+
+        renumbered_res_dict[chain_id] = []
+
+        for res in res_dict[chain_id]:
+
+            res = renumber_chain_res_num(res, chain_id, af_offset)
```

Ruff (0.8.2)

Undefined name `renumber_chain_res_num`

(F821)